README.md

```diff
@@ -1,3 +1,25 @@
-# myDiary
-myDairy is an online journal that allows users to make daily journal entries.
-The site is published at https://sammuel09.github.io/myDiary/
+# Project Title
+myDiary is a web-based journal application. 
+
+## Getting Started
+In order to have a copy of this project, clone this project to your local repository.
+
+### Prerequisites
+You need to have NodeJs installed on yourlocal machine. Install Node [here](https://nodejs.org/en/)
+
+### Frameworks Used
```

Comment:
You should update this section when you add database and other tools. nice job

Reply:
Yes I will.